intl/msg_hash_us.h

```diff
@@ -763,6 +763,24 @@ MSG_HASH(MENU_ENUM_LABEL_VALUE_INPUT_JOYPAD_Y,
       "Y button (left)")
 MSG_HASH(MENU_ENUM_LABEL_VALUE_INPUT_KEY,
       "(Key: %s)")
+MSG_HASH(MENU_ENUM_LABEL_VALUE_INPUT_MOUSE_LEFT,
+      "MOUSE1")
```

Comment:
Should be probably Mouse 1 to match Key strings and Button strings

Reply:
Happy to rename these to fit in better, etc.  
Mouse 1, Mouse 2, ... 
Wheel Up, Wheel Down, Wheel Left, Wheel Right, ?

Reply:
Yeah